pkg/helmx/option.go

```diff
@@ -0,0 +1,180 @@
+package helmx
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
+	cpb "k8s.io/helm/pkg/proto/hapi/chart"
+	rls "k8s.io/helm/pkg/proto/hapi/services"
+)
+
+// # Options
```

Comment:
Tiny nit: type comments should follow the same convention as functions: `Option allows specifying...`.